doc/man1/openssl-s_server.pod.in

```diff
@@ -379,28 +379,30 @@ DH).
 
 Inhibit printing of session and certificate information.
 
+=item B<-tlsextdebug>
+
+Print a hex dump of any TLS extensions received from the server.
+
 =item B<-www>
 
 Sends a status message back to the client when it connects. This includes
 information about the ciphers used and various session parameters.
-The output is in HTML format so this option will normally be used with a
-web browser. Cannot be used in conjunction with B<-early_data>.
+The output is in HTML format so this option can be used with a web browser.
+The special URL C</renegcert> turns on client cert validation, and C</reneg>
+tells the server to request renegotiation.
+The B<-early_data> option cannot be used with this option.
 
-=item B<-WWW>
+=item B<-WWW>, B<-HTTP>
 
 Emulates a simple web server. Pages will be resolved relative to the
-current directory, for example if the URL https://myhost/page.html is
-requested the file F<./page.html> will be loaded.
-The files loaded are
+current directory, for example if the URL C<https://myhost/page.html> is
+requested the file F<./page.html> will be loaded, if a directory is specified
+a simple listing will be returned, and so on. Any files loaded are
```

Comment:
I am sorry but this is still not completely right. The "... are assumed to contain complete and correct HTTP response" applies only to `B<-HTTP>`. In case of B<-WWW> the response is (in very simple way) generated by the s_server itself.

Reply:
Fixup commit pushed.  Did you trace the code, or did you just know this from experience? :) I'm not sure that merging the two makes sense any more, what do you think?

Reply:
I looked at the source. This was actually documented in the original docs.

Reply:
Both merged and non-merged works for me. I'll approve this version, if you like to separate them again, no problem.

Reply:
I'm going to leave well enough alone, since you think it's okay as-is.